index.js

```diff
@@ -41,33 +58,41 @@ async function work() {
     httpsAgent: https.Agent({rejectUnauthorized}),
   });
   if(send) {
-    return _sendOperation({ledgerClient});
+    const record = getRecord(recordPath);
+    return _sendOperation({ledgerClient, record});
   }
   if(getRecordId) {
     return _getRecord({id: getRecordId, ledgerClient});
   }
 }
 
 async function _getRecord({id, ledgerClient}) {
-  const result = await ledgerClient.getRecord({id});
-  console.log(`Record details:\n${JSON.stringify(result, null, 2)}`);
+  try {
+    const result = await ledgerClient.getRecord({id});
+    console.log(`Record details:\n${JSON.stringify(result, null, 2)}`);
+  } catch(e) {
+    console.error(`command get record failed for ${id}.`);
+    console.error(e);
+  }
 }
 
-async function _sendOperation({ledgerClient}) {
-  const creator = await ledgerClient.getTargetNode();
-  const operation = {
-    '@context': constants.WEB_LEDGER_CONTEXT_V1_URL,
-    creator,
-    type: 'CreateWebLedgerRecord',
-    record: {
-      '@context': constants.TEST_CONTEXT_V1_URL,
-      id: `https://example.com/transaction/${uuid()}`,
-      price: Math.floor(Math.random() * 1000000000000),
-    }
-  };
-  await ledgerClient.sendOperation({operation});
-  console.log(
-    `Operation sent successfully.\n${JSON.stringify(operation, null, 2)}`);
+
+async function _sendOperation({ledgerClient, record}) {
+  try {
+    const creator = await ledgerClient.getTargetNode();
+    const operation = {
+      '@context': constants.WEB_LEDGER_CONTEXT_V1_URL,
+      creator,
+      type: 'CreateWebLedgerRecord',
+      record
+    };
+    await ledgerClient.sendOperation({operation});
+    console.log(
+      `Operation sent successfully.\n${JSON.stringify(operation, null, 2)}`);
+  } catch(e) {
+    console.error('command send record failed.');
```

Comment:
combine with logging immediately below.

Reply:
Addressed in https://github.com/digitalbazaar/bedrock-ledger-test-cli/pull/1/commits/e46fb1ced35923628ce105d70f27de6ecf2bd4cd